core/compatibility/components/blocks/Category/Sidebar.js

```diff
@@ -24,7 +26,7 @@ export default {
       this.$bus.$emit('filter-reset')
       this.$store.dispatch('category/resetFilters')
       this.$store.dispatch('category/searchProductQuery', buildFilterProductsQuery(this.category, this.activeFilters))
-      this.$store.dispatch('category/products', this.$store.state.category.current_product_query)
+      this.$store.dispatch('category/products', this.getCategorySearchOptions)
```

Comment:
Should be getCategoryCurrentProductQuery

Reply:
ok thanks - i was wondering how to approach this. I havn't name it `query` just because it's not string but object; but i thought that i'll commit that and ask today for advice :)

but again - not sure about that naming convention. It's start of big store  refactor and i think we should discuss it first:)